src/api/index.js

```diff
@@ -4,6 +4,7 @@ import copy from '../copy/en'
 
 export default ({ config, controller }) => {
   let api = Router()
+  const fileDest = config.EXPORT_FILE_DEST || ""
```

Comment:
What is the logic behind the default here being an empty string? Would it be better as 'data_export.json' or something similarly generic?

Reply:
Ah I see down below that the empty string is being used as a 'null' value. Maybe we explicitly use `null` instead?